lib/transport/datagouvfr/authentication.ex

```diff
@@ -36,4 +47,14 @@ defmodule Transport.Datagouvfr.Authentication do
     |> put_header("accept", "application/json")
     |> AuthCode.get_token(params, headers)
   end
+
+  defmacro __using__(_) do
+    quote do
+      alias Transport.Datagouvfr.Authentication
```

Comment:
Useless ?

Reply:
nop, if I don't do that I can't make `mix credo --strict` work